applications/machine_learning/README.rst

```diff
@@ -207,6 +207,9 @@ Buttons
 The application supports a button that is used to switch between data forwarding and running the machine learning model.
 You can trigger the change by pressing and holding the button for longer than 5 seconds.
 
+.. note::
+   If a given configuration supports Bluetooth, then pressing the button shortly after application boot triggers removing Bluetooth bond.
```

Comment:
```suggestion
   If a given configuration supports Bluetooth, the Bluetooth bond can be removed by pressing a button shortly after the application has booted.
```
I think we should mention which button will remove the bond (like it a reset button or something else)

Reply:
The button used to switch between data forwarding and running the machine learning model. It's actually configurable and it may depend on used board.